news/2023/2023-05-01-vocaloid-mapping-contest.md.md

```diff
@@ -0,0 +1,113 @@
+
+The [**Vocaloid Mapping Contest (VMC)**](/wiki/Contests/VMC) is all about finding and promoting VOCALOID, UTAU, and other vocal synth songs and artists that have fallen through the cracks. Our [prototype contest last year](https://osu.ppy.sh/community/forums/topics/1604262?n=1) was themed around Hatsune Miku for her 15th birthday, while this contest has expanded to a variety of fun VOCALOID songs!
+
+Your goal for this contest: **use [one of these songs](#songs) to create the best VOCALOID beatmap possible** (as determined by our [judging panel](osu.ppy.sh/wiki/contests/VMC/1#Staff_and_Judges)).
+
+And to make things more interesting, a special guest judge has joined us: **legendary VOCALOID mapper [*tutuhaha*](https://osu.ppy.sh/users/546991)**!
+
+Let's break everything down.
+
+## Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Announcement | 2023-05-01 |
+| Submission | 2023-05-01 to 2023-05-31 |
+| Judging | 2023-06-01 to 2023-06-24 |
+| Results | 2023-07-01 |
+
+## Prizes
+
+What kind of contest doesn't have prizes! We have to reward you for your hard work in some way, yeah?
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| 1st place | 6 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points), *VMC Winner* profile badge, assistance in the ranking process |
+| 2nd place | 2 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |
+| 3rd place | 1 month of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |
+
+This contest is created by and for vocal synth fans to elevate the genre (and those who map it) to new heights. All mappers who participate will receive constructive feedback from some of the most well-known vocal synth mappers in the community, and that's a prize in itself!
+
+## Rules
+
+Believe it or not, contests have rules to keep everything fair.
+
+- **Submissions must be for the osu! ruleset only.** Comparing beatmaps of different modes in a judging scenario is unfortunately not realistic.
+- **Submissions must use one of the [selected songs](#songs)**
+- **Submissions must be in `.osz` format.**
+- **Beatmaps should abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes, submissions that could not be ranked without major changes will be disqualified.
+- **Submissions must not be publicly revealed until after the results are announced.**
+- **Excluding metadata, submissions must not include anything that could reveal your identity.** Metadata will be anonymised before submissions are relayed to judges.
+- **Collaborations are not allowed**. This is a solo contest.
+- **Submissions must not include storyboards or videos.**
+- **Do not edit any settings in the .osu that would not be possible through the editor itself.** You are permitted to change any difficulty settings, backgrounds, green lines, SV, hitsounds, etc.
+- **You must submit only one difficulty.** Only one entry is allowed.
```

Comment:
```suggestion
- **Your submission must have only one difficulty.**
```
if you're using the osu contest listing, it's implied that only one submission is allowed anyway